src/Transport/Topology/Topologies/ForwardingTopologySectionManager.cs

```diff
@@ -273,16 +283,21 @@ void BuildTopicBundles(RuntimeNamespaceInfo[] namespaces, AddressingLogic addres
             var numberOfEntitiesInBundle = settings.Get<int>(WellKnownConfigurationKeys.Topology.Bundling.NumberOfEntitiesInBundle);
             var bundlePrefix = settings.Get<string>(WellKnownConfigurationKeys.Topology.Bundling.BundlePrefix);
 
-            for (var i = 1; i <= numberOfEntitiesInBundle; i++)
+            foreach (var @namespace in namespaces)
             {
-                topics.AddRange(namespaces.Select(n => new EntityInfo
+                var numberOfTopicsFound = namespaceBundleConfigurations.Value.GetNumberOfTopicInBundle(@namespace.Alias);
```

Comment:
So the implications here are that the first materialization of `.Value` will take time and after the first call the result will be cached.